OpenTreeMap/src/OTM/Controllers/OTMFilterListViewController.m

```diff
@@ -481,20 +500,13 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
     return [[newText componentsSeparatedByString:@"."] count] <= 2;
 }
 
-- (UIView *)createView {
-    CGRect r = CGRectMake(0,0,320,55);
-    [self setView:[[UIView alloc] initWithFrame:r]];
-
-    CGFloat padding = 10.0f;
+- (void)addSubviews {
+    CGFloat padding = 10;
 
-    CGRect nameFrame = CGRectMake(21,0,320,50);
-    CGRect leftFrame = CGRectMake(135,10,65,31);
+    CGRect leftFrame = CGRectMake(0,0,65,31);
```

Comment:
If we change the width from `65` to `122` the text boxes on the range filters will match the date range filters.

<img width="321" alt="screen shot 2016-02-11 at 10 57 50 am" src="https://cloud.githubusercontent.com/assets/17363/12985159/58414b82-d0ae-11e5-8dc3-5a32500feedd.png">

<img width="321" alt="screen shot 2016-02-11 at 10 55 35 am" src="https://cloud.githubusercontent.com/assets/17363/12985164/5c7520de-d0ae-11e5-85c4-e7d96fd42e85.png">